### ISSUE TITLE
View Picker Disappear after select type

### ISSUE THREAD

**Issue by Us3r-gitHub** [twentyhq/twenty#6168]
## Bug Description

https://github.com/twentyhq/twenty/assets/58467104/9bdcfb79-22de-45ea-80f9-68d70f4621b7

## Expected behavior
View Picker should be keep visible and adds a new field `select` when kanban is selected

**Comment by FelixMalfait** [comment]
This is strange, I'm not able to reproduce it. I've tried different scenarios, with an object that has a select field, an object without a select field, and it always works for me 🤔. Would you have any idea what could be specific to your use-case? Does it happen on all objects? Thanks

**Comment by Us3r-gitHub** [comment]
I'm not sure what I'm doing

#### Historical timeline
1. I want to work on issue #5661 
2. So I created new branch referencing this commit (7aa903aa5)
I tried version 0.21 (1c3ea9b10) and gave me same bug, maybe you could try it with this version and let me know if you see the same behavior.

#### Note:
Please run the database reset before starting the server.

**Comment by Us3r-gitHub** [comment]
Currently, in latest commit (6bc36635e), I'm facing another issue

![image](https://github.com/twentyhq/twenty/assets/58467104/00268fce-7a10-4260-a79e-6c74c2f8bd4a)
DB reset is broken (seeder is not running successfully)

![image](https://github.com/twentyhq/twenty/assets/58467104/7fbdc913-0ffe-4783-aed6-1ccefcb41806)
As a result, when I run server and open application, there is no dummy data

**Comment by FelixMalfait** [comment]
@Us3r-gitHub I think this was fixed, if you merge main it should work!
